File: Arreglos/RecorridoArraysMatrices/RecorridoI/RecorridoI.cs
// Recorrido Inverso de un Arreglo en C#
// Autor: Rivera Cota Jesús Crispín
// FACULTAD DE INGENIERÍA LOS MOCHIS
// Ing. de Software 201

using System;

class Program {
    static void Main() {
        int[] arr = { 40, 50, 60, 70, 80, 90 };

        Console.Write("Recorrido inverso del array: ");
        Console.Write("\nLos elementos del array son: ");
        for (int idx = arr.Length - 1; idx >= 0; idx--) {
            Console.Write(arr[idx] + " ");
        }
        Console.WriteLine();
    }
}

File: Arreglos/RecorridoArraysMatrices/RecorridoI.cpp
// Recorrido Inverso de un Arreglo en C++
// Autor: Rivera Cota Jesús Crispín
// FACULTAD DE INGENIERÍA LOS MOCHIS
// Ing. de Software 201

#include <iostream>
using namespace std;

int main() {
    int arr[] = {40, 50, 60, 70, 80, 90};
    int size = sizeof(arr) / sizeof(arr[0]);

    cout << "Recorrido inverso del array: ";
    cout << "\nLos elementos del array son: ";
    for (int idx = size - 1; idx >= 0; idx--) {
        cout << arr[idx] << " ";
    }
    cout << endl;

    return 0;
}